source/clustering.md

```diff
@@ -632,6 +630,10 @@ visualize them as shown in {numref}`cluster_plot`.
 Note that we are plotting the *un-standardized* data here; if we for some reason wanted to 
 visualize the *standardized* data, we would need to use the `fit` and `transform` functions
 on the `StandardScaler` preprocessor directly to obtain that first.
+As in Chapter {ref}`viz`,
```

Comment:
I added a reminder here so that students remember since the viz chapter was a while ago